common-lib/src/cmr/common/util.clj

```diff
@@ -198,40 +198,6 @@
                        "Timed function %s/%s took %d ms."
                        ~ns-str ~fn-name-str elapsed#)))))))))))
 
-(defn ^{:deprecated true} build-validator
-  "Creates a function that will call f with it's arguments. If f returns any
-  errors then it will throw a service error of the type given.
-
-  DEPRECATED: we should use the validations namespace."
```

Comment:
I believe this is intended for the cmr.common.validations namespace, not the cmr.umm-spec.validation

Reply:
I removed the need for that and moved this into what library I could that was shared between metadata-db and search. I'm not sure why it was deprecated.